lprnet/LPRnet.cpp

```diff
@@ -399,8 +399,19 @@ int main(int argc, char **argv) {
     cv::resize(img, pr_img, cv::Size(INPUT_W, INPUT_H), 0, 0, cv::INTER_CUBIC);
     // For multi-batch, I feed the same image multiple times.
     // If you want to process different images in a batch, you need adapt it.
-   cv::Mat blob = cv::dnn::blobFromImage(pr_img, 0.0078125, pr_img.size(), cv::Scalar(127.5, 127.5, 127.5), true,
-                                          false);
+   //cv::Mat blob = cv::dnn::blobFromImage(pr_img, 0.0078125, pr_img.size(), cv::Scalar(127.5, 127.5, 127.5), true,
+                                          //false);
+    int i = 0;
+    for (int row = 0; row < INPUT_H; ++row) {
+        uchar* uc_pixel = pr_img.data + row * pr_img.step;
+        for (int col = 0; col < INPUT_W; ++col) {
+            data[i + 2 * INPUT_H * INPUT_W] = ((float)uc_pixel[2] - 127.5)*0.0078125;
+            data[i + INPUT_H * INPUT_W] = ((float)uc_pixel[1]-127.5)*0.0078125;
+            data[i] = ((float)uc_pixel[0]-127.5)*0.0078125;
+            uc_pixel += 3;
+            ++i;
+        }
+    }
```

Comment:
Do you mean the input should be B G R not R G B?

Reply:
the point is [https://github.com/xuexingyu24/License_Plate_Detection_Pytorch/blob/master/LPRNet/LPRNet_Test.py](url)  88 line np.float then transpose the normalized from -1 to 1  is differenet from the code in [https://github.com/wang-xinyu/tensorrtx/blob/master/lprnet/LPRnet.cpp](url) in 402 line blobFromImage; update my process, the value  input  tensorrt lpr model is equal to  [https://github.com/xuexingyu24/License_Plate_Detection_Pytorch/blob/master/LPRNet/LPRNet_Test.py](url)  input the lpr pytorch model;  please check